morf-core/src/main/java/org/alfasoftware/morf/metadata/SchemaUtils.java

```diff
@@ -562,6 +591,15 @@ public TableBuilder indexes(Index... indexes) {
     }
 
 
+    /**
+     * @see org.alfasoftware.morf.metadata.SchemaUtils.TableBuilder#indexes(java.lang.Iterable)
+     */
+    @Override
+    public TableBuilder indexes(Iterable<? extends Index> indexes) {
+      return new TableBuilderImpl(getName(), columns(), indexes, isTemporary());

```

Comment:
Can we make indexes(Index... indexes) call this method since we construct a list in it.

Reply:
👍